lib/config.js

```diff
@@ -154,6 +154,8 @@ function getLocalConfig(thisConfig, directory) {
 
         if (personalConfig) {
             return ConfigOps.merge(config, personalConfig);
+        } else if (thisConfig.options.rules && thisConfig.options.rules !== {}) {
```

Comment:
Doh, I wondered about that.  I want to check that it's not empty.  So maybe `Object.keys(thisConfig.options.rules).length > 0` would work?